charmcraft.yaml

```diff
@@ -4,10 +4,16 @@
 # See https://juju.is/docs/sdk/charmcraft-config for guidance.
 
 type: charm
-bases:
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-    run-on:
-    - name: ubuntu
-      channel: "22.04"
```

Comment:
The `bases` does not work anymore if only `24.04` is specified. Have to use the new `base`, `platforms` options from charmcraft 3.